kolibri/core/analytics/middleware.py

```diff
@@ -79,7 +79,9 @@ def middleware(request):
         # Silence busy polling API endpoints and PATCH requests:
         # https://github.com/learningequality/kolibri/issues/6459
         log_as_debug = (
-            "/api/tasks/tasks/" in request.path_info or request.method == "PATCH"
+            "/api/tasks/tasks/" in request.path_info
+            or "/status/" in request.path_info
```

Comment:
Since `/status/` is a bit less specific than the other `/api/tasks/tasks` path, perhaps the comparison should be stricter?

Reply:
ah makes sense to me! I will make the change. Thank you Blaine!